pkg/dependency/dotnet_sdk_test.go

```diff
@@ -323,6 +323,7 @@ func testDotnetSDK(t *testing.T, when spec.G, it spec.S) {
 				SHA:             "some-sha256",
 				ReleaseDate:     "2020-02-20T00:00:00Z",
 				DeprecationDate: "2050-02-20T00:00:00Z",
+				CPE:             "cpe:2.3:a:microsoft:.net_core:2.0.201:*:*:*:*:*:*:*",
```

Comment:
I haven't been able to find any CPE results for Dotnet SDK versions or any CVE's that reference a CPE like this (with an SDK version number). Did you see this notation somewhere?

Reply:
https://nvd.nist.gov/products/cpe/detail/842688?namingFormat=2.3&orderBy=CPEURI&keyword=cpe%3A2.3%3Aa%3Amicrosoft%3A.net_core&status=FINAL

Reply:
There are no CVE's for that CPE but it seems like it's a legit CPE so seems like we can use it. 

It does look like we'll need to use `.net` instead of `.net_core` for versions `5.*.*` and up, though.

Reply:
Fixed in latest commit